raw_syllabus.md

```diff
@@ -0,0 +1,42 @@
+# ICME Python Summer Workshop, 2023 edition
```

Comment:
Is this file still needed?

Reply:
Do not need the `raw_syllabus.md` anymore, do you want to remove it?

Reply:
Yeah let's nuke it.